service/soa_service.go

```diff
@@ -22,8 +22,8 @@ func ValidateStatementDetails(svc dao.Service, statementDao *models.StatementOfA
 	}
 
 	// Check that the attachment has been submitted correctly
-	if len(statementDao.Attachments) == 0 || len(statementDao.Attachments) > 1 {
-		errs = append(errs, "please supply only one attachment")
+	if len(statementDao.Attachments) == 0 || len(statementDao.Attachments) > 2 {
+		errs = append(errs, "please supply at least one attachment ID (up to a maximum of two attachment IDs)")
 	}
 
```

Comment:
If more than two attachments are submitted, the error response could be misleading to say "please supply at least one attachment". Better to split the checks for nil case and more than 2 with clearer error responses.